src/main/java/com/epam/service/TourOfferService.java

```diff
@@ -94,7 +95,12 @@ else if(tourDescription == null){
         for(Hotel hotel: myList){
             listOfHotelsId.add(hotel.getId());
         }
-            return tourOfferDAO.searchTours(listOfHotelsId, startDate, endDate);
+         List<TourOffer> listOfTours = tourOfferDAO.searchTours(listOfHotelsId, startDate, endDate);
+        if(listOfTours.size() == 0){
+            log.error("No tours available");
```

Comment:
I believe it is not error, just a regular case. U can use info/warning log level here

Reply:
Yes, you right.

Reply:
Changed level of log to "warn".